src/index.html

```diff
@@ -188,5 +188,47 @@ <h1>Made with love.</h1>
       ga('create', 'UA-67015100-1', 'auto');
       ga('send', 'pageview');
     </script>
+    <a href="https://github.com/get-alex/alex" class="github-corner" aria-label="View source on GitHub"><svg
```

Comment:
Could you indent this nicely, so that the SVG is on a newline, indented one leve, compared to the link?